src/managers/input/InputManager.cpp

```diff
@@ -545,11 +545,25 @@ void CInputManager::processMouseDownNormal(wlr_pointer_button_event* e) {
     if (!PASS && !*PPASSMOUSE)
         return;
 
+    const auto mouseCoords = g_pInputManager->getMouseCoordsInternal();
+    const auto w           = g_pCompositor->vectorToWindowIdeal(mouseCoords);
+
+    if (w && !w->m_bIsFullscreen && !w->hasPopupAt(mouseCoords) && w->m_sGroupData.pNextWindow) {
+        const wlr_box box = w->getDecorationByType(DECORATION_GROUPBAR)->getWindowDecorationBox().getExtents();
```

Comment:
you made a CRegion::containsPoint for this exact reason no?

Reply:
I need the box info to calculate what window should be selected (for example)
no point in using the CRegion::containsPoint since I would still need to get the `box.x` and `box.width`

ideal solution would be for each window's tab to be it's own pixman box all making up the pixman region
that way something like `CRegion::boxIndex(const Vector2D&)` would give the needed group window index

other solution is each window would be responsible for its own group tab (wouldn't be hidden), which all together would make up the full bar

Reply:
hm, fair point. Alright.